### PR TITLE
Add device types1

### PR DESCRIPTION
**Pull request by jvanderaa**

### CODE REVIEW

File: cisco/asr1002-hx.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+u_height: 2
+is_full_depth: false
+console_ports:
+  - name: con 0
+power_ports:
+  - name: P0
+  - name: P1
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: SFP (1GE)
```
Comment (jeremystretch) [inline cisco/asr1002-hx.yaml]: Interface types must reference the new (v2.7+) slugs; see the [schema validation](https://github.com/netbox-community/devicetype-library/blob/master/tests/schema.json) for all valid values.

(This should have caused the automated tests to fail, but we weren't catching `.yml` suffixes. This has been fixed.)

File: cisco/asr1002-hx.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    mgmt_only: false
```
Comment (jeremystretch) [inline cisco/asr1002-hx.yaml]: `mgmt_only` is False by default; you can omit this line from non-management interfaces for brevity.

File: cisco/asr1002-hx.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+manufacturer: Cisco
+model: ASR1002-HX
+slug: asr1002-hx
+u_height: 2
+is_full_depth: false
```
Comment (jeremystretch) [inline cisco/asr1002-hx.yaml]: All devices with front-to-back airflow should be considered full-depth. (Installing a device in the same unit on the opposite face would inhibit airflow.)

**Comment by jeremystretch** [comment]
Also, please use `Cisco` for the manufacturer directory (capital C).

File: cisco/asr1002-hx.yaml (hunk 3 above)
Comment (jvanderaa) [reply cisco/asr1002-hx.yaml]: Set this to true.

File: cisco/asr1002-hx.yaml (hunk 2 above)
Comment (jvanderaa) [reply cisco/asr1002-hx.yaml]: Updated on all the interfaces, should be looking better.

File: cisco/asr1002-hx.yaml (hunk 1 above)
Comment (jvanderaa) [reply cisco/asr1002-hx.yaml]: Makes sense. Sorry for the PR in that setup. Went too quick.

**Comment by jeremystretch** [comment]
@jvanderaa Looks like you need to update your fork. There should be a `device-types` directory that holds all the manufacturers. Once you've done that, the validation tests should run as expected.

**Comment by jeremystretch** [comment]
This PR is superseded by #16

**Pull request closed without merging**